flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/stream/table/TableSinkITCase.scala

```diff
@@ -1361,20 +1361,16 @@ class TableSinkITCase extends StreamingTestBase {
 
   @Test
   def testExecuteInsertToTableDescriptor(): Unit = {
-    val schema = Schema.newBuilder()
-      .column("f0", DataTypes.INT())
-      .build();
-
     val tableId = TestValuesTableFactory.registerData(Seq(row(42)))
     tEnv.createTemporaryTable("T", TableDescriptor.forConnector("values")
-      .schema(schema)
+      .schema(Schema.newBuilder()
+        .column("f0", DataTypes.INT())
+        .build())
       .option("data-id", tableId)
       .option("bounded", "true")
       .build())
 
-    val tableResult = tEnv.from("T").executeInsert(TableDescriptor.forConnector("values")
-      .schema(schema)
-      .build())
+    val tableResult = tEnv.from("T").executeInsert(TableDescriptor.forConnector("values").build())
```

Comment:
we definitely need more tests here:
- one with explicit schema
- one with primary key and metadata columns but not physical columns
- one with everything derived

Reply:
The reason I went with this is because it tests the entire path end-to-end, and then relies on existing tests of the schema translator. I'm happy to add explicit test cases for all of these here, but that'll be a bunch of IT cases which is what I was trying to avoid this way?

Reply:
You are right. Adding more IT cases should be avoided. But adding explicit test cases should be the way to go.

Reply:
I extended the IT case for now, but it still adds executions.